lib/gateways/referral-gateway.js

```diff
@@ -52,7 +52,8 @@ class ReferralGateway {
 
     const request = {
       TableName: this.tableName,
-      ProjectionExpression: 'id, firstName, lastName, statusHistory',
+      ProjectionExpression:
+        'id, firstName, lastName, statusHistory, referralNumber, referrer, resident, service',
```

Comment:
Should this be referenceNumber? Assuming it's after the nano(5) value in the DB

Reply:
Yes! Nice spot